doc/Installation/Installation-Ubuntu-2004-Apache.md

```diff
@@ -0,0 +1,245 @@
+source: Installation/Installation-Ubuntu-2004-Apache.md
+path: blob/master/doc/
+
+> NOTE: These instructions assume you are the **root** user.  If you
+> are not, prepend `sudo` to the shell commands (the ones that aren't
+> at `mysql>` prompts) or temporarily become a user with root
+> privileges with `sudo -s` or `sudo -i`.
+
+**Please note the minimum supported PHP version is 7.2.5**
+
+# Set system timezone
+Set the system timezone if it hasn't already been set.
+This timezone must match the timezone used later in the PHP configuration.
+To list available timezones, run `timedatectl list-timezones`.
+Replace `America/New_York` with your correct timezone.
+```bash
+timedatectl set-timezone America/New_York
+```
+
+# Install Required Packages
+
+```bash
+apt update
+apt install acl curl apache2 composer fping git graphviz imagemagick libapache2-mod-php7.4 mariadb-client mariadb-server mtr-tiny nmap php7.4-cli php7.4-curl php7.4-gd php7.4-json php7.4-mbstring php7.4-mysql php7.4-snmp php7.4-xml php7.4-zip rrdtool snmp snmpd whois python3-pip

```

Comment:
Ubuntu 20.04 does not need pip, it has new enough python3 libraries in it's repo and you should install them from there.

Reply:
Thank you. I have updated the list of packages.